api/swagger.yml

```diff
@@ -1134,6 +1144,25 @@ paths:
           default:
             $ref: "#/components/responses/ServerError"
 
+  /auth/capabilities:
+    get:
+      tags:
+        - auth
+      operationId: getAuthCapabilities
+      summary: list authentication capabilities supported
+
+      responses:
+        200:
+          description: auth capabilities
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/AuthCapabilities"
+        401:
```

Comment:
@guy-har, not sure we need 401 here as this endpoint must be public as it serves for reset password as well, which at this stage the user is unauthenticated.